defaults/main.yml

```diff
@@ -4,7 +4,9 @@ zookeeper_playbook_version: "0.9.2"
 zookeeper_version: 3.4.6
 zookeeper_url: http://www.us.apache.org/dist/zookeeper/zookeeper-{{zookeeper_version}}/zookeeper-{{zookeeper_version}}.tar.gz
 
+zookeeper_debian_apt_install: false
```

Comment:
I agree.
ps: I suggest to update the documentation a bit (explaining this new setting)
In general I like to have each role variable have a 1-line documentation